frontend/run.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+if [ ${ENV} == "PROD" ]; then 
+    yarn run build
+    cp -R ./public/. ./build/
```

Comment:
can you explain why you are copying the files in public to build? Doesn't build/static already have all the files needed?

Reply:
"build" is the shared volume with nginx container

Strictly speaking, we can just mount "public" as the shared volume. I did it in this way to make everything clean

Reply:
What files in public are not in build already?

Reply:
Yes you are right. I missed that point